tests/test_filters.py

```diff
@@ -776,6 +775,16 @@ def test_per_extension_filter(self):
             expected_data, filtered_data
         ), f"Expected {expected_data} but got {filtered_data}"
 
+        tests = [["", (0, 0.0)],
```

Comment:
Can you use `@pytest.mark.parametrize` for these tests instead of writing the for loop? You can take a look at the fuzzy dedup tests [here](https://github.com/NVIDIA/NeMo-Curator/blob/main/tests/test_fuzzy_dedup.py#L118) for pointers, it's not too big of a change.

Reply:
Done!